src/stores/UserStore.ts

```diff
@@ -40,15 +42,18 @@ const userStoreLocalStorage = z.object({
   refresh: z.string(),
   env: z.string(),
   whoami: whoamiResponse.nullable(),
+  jiraUsername: z.string(),
```

Comment:
I think this can be good scenario to create a user model with other fields we get from Jira.

Reply:
I added a `zodJira.ts` file on my PR https://github.com/RedHatProductSecurity/osim/pull/306/files#diff-4649669d92d9bc235fd085e35b7e1f000b295a3c5c1737007ab5e0a62b383d0a with the responses of the endpoints I use, I think it makes sense to type the `getJiraUserInfo` as well